build.gradle

```diff
@@ -95,35 +85,30 @@ subprojects {
     }
 
     dependencies {
-
         common 'joda-time:joda-time:2.3',
-                'com.google.inject:guice:4.0',
+                'com.google.inject:guice:5.0.1',
                 "javax.xml.bind:jaxb-api:2.3.1",
                 "com.sun.xml.bind:jaxb-core:2.3.0.1",
-                "com.sun.xml.bind:jaxb-impl:2.3.0.1"
+                "com.sun.xml.bind:jaxb-impl:2.3.0.1",
+                "org.glassfish.jersey.media:jersey-media-jaxb:2.33"
 
         dropwizard "io.dropwizard:dropwizard-core:$dependencyVersions.dropwizard",
                 "io.dropwizard:dropwizard-client:$dependencyVersions.dropwizard",
                 "io.dropwizard:dropwizard-auth:$dependencyVersions.dropwizard",
                 "io.dropwizard:dropwizard-configuration:$dependencyVersions.dropwizard",
                 "io.dropwizard:dropwizard-metrics-graphite:$dependencyVersions.dropwizard",
-                'com.hubspot.dropwizard:dropwizard-guicier:1.0.0.6',
+                'ru.vyarus:dropwizard-guicey:5.3.0',
                 'org.reflections:reflections:0.9.10',
-                'org.dhatim:dropwizard-sentry:1.3.9-2'
+                'org.dhatim:dropwizard-sentry:2.0.20'
 
         config 'commons-io:commons-io:2.1'
 
-        verify_event_emitter "uk.gov.ida:verify-event-emitter:2.0.0-82"
+        verify_event_emitter "uk.gov.ida:verify-event-emitter:2.0.0-SNAPSHOT"
```

Comment:
I've been a bit inconsistent here - but you've acknowledged elsewhere that `SNAPSHOT` references will change to real versions anyway when the time comes.